hack/cluster-sync-handler.sh

```diff
@@ -10,19 +10,22 @@ sed "s#--v=production#--v=debug#" ${local_handler_manifest} | ${KUBECTL} create
 
 for i in {300..0}; do
     # We have to re-check desired number, sometimes takes some time to be filled in
-    desiredNumberScheduled=$(${KUBECTL} get daemonset -n nmstate nmstate-handler -o=jsonpath='{.status.desiredNumberScheduled}')
+    desiredMasterHandlersNumberScheduled=$(${KUBECTL} get daemonset -n nmstate nmstate-handler -o=jsonpath='{.status.desiredNumberScheduled}')
+    desiredWorkerHandlersNumberScheduled=$(${KUBECTL} get daemonset -n nmstate nmstate-handler-worker -o=jsonpath='{.status.desiredNumberScheduled}')
 
-    numberAvailable=$(${KUBECTL} get daemonset -n nmstate nmstate-handler -o=jsonpath='{.status.numberAvailable}')
+    numberAvailableMasterHandlers=$(${KUBECTL} get daemonset -n nmstate nmstate-handler -o=jsonpath='{.status.numberAvailable}')
+    numberAvailableWorkerHandlers=$(${KUBECTL} get daemonset -n nmstate nmstate-handler-worker -o=jsonpath='{.status.numberAvailable}')
 
-    if [ "$desiredNumberScheduled" == "$numberAvailable" ]; then
-        echo "nmstate-handler DS is ready"
+    if [ "$desiredMasterHandlersNumberScheduled" == "$numberAvailableMasterHandlers" ] && [ "$desiredWorkerHandlersNumberScheduled" == "$numberAvailableWorkerHandlers" ]; then
+        echo "nmstate-handler and nmstate-handler-worker DSs are ready"
         break
     fi
 
     if [ $i -eq 0 ]; then
```

Comment:
If we refactor this a little we can have something nice like 

```bash
if isOk "nmstate-handler-worker" && isOk "nmstate-handler"; then ...
```
So we will get the specific daemonset at logs.